### PR TITLE
Translation defaults

### LINKED ISSUE #238: Translate the interface and connect it with Crowdin
The first step is to add all texts in the file "src/locale/default.json", map the text with $t and $tc to translate them. Example:

default.json
```
{
  "createSpace": "Create space"
  "spaceCount": "{0} space(s)"
}
```
view
```
<template>
  {{ $t('createSpace') }} {{ $tc('spaceCount', [count]) }}
</template>
```

### PR DESCRIPTION
**Pull request by samuveth**
Fixes #238

### CODE REVIEW

File: src/locales/default.json [hunk 1]
```diff
@@ -1,3 +1,70 @@
 {
-  "createSpace": "Create space"
+  "searchPlaceholder": "Search",
+  "spaces": "space(s)",
```
Comment (bonustrack) [inline src/locales/default.json:3]: We should always store the complete sentence here, for such text you can add variable like this: 
```suggestion
  "spaces": "{0} space(s)",
```

Comment (samuveth) [reply src/locales/default.json:3]: Oh I see

**Comment by samuveth** [comment]
@bonustrack It's not done yet, I will do one more run tomorrow to make sure everything in included.

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**